Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -67,34 +68,44 @@ def test_cohesity_close_helios_incident(self):
         print("alert_id --> %s" % alert_id)
         print("api_key --> %s" % self.api_key)
         self.assertEqual(alert_details['alertState'], "kSuppressed")
-
-    def test_all_incidents_in_helios(self):
-        """
-        This test case verifies that all incidents in the Sentinel workspace have corresponding alerts in Helios, and
-        that the number of alerts in Helios matches the number of incidents in the Sentinel workspace.
-        """
-        ids = get_incident_ids(self.resource_group, self.workspace_name)
-        alert_ids = [alert_id for (vid, alert_id) in ids]
-        for alert_id in alert_ids:
-            self.assertIsNotNone(get_alert_details(alert_id, self.api_key), f"alert_id --> {alert_id} doesn't exist in helios.")
-        alerts_details = get_alerts_details(alert_ids, self.api_key)
-        self.assertEqual(len(alert_ids), len(alerts_details))
-
-    def test_no_dup_incidents(self):
-        """
-        This test case verifies that there are no duplicate incidents in the Sentinel workspace.
-        """
-        ids = get_incident_ids(self.resource_group, self.workspace_name)
-        alert_ids = [alert_id for (vid, alert_id) in ids]
-        return len(alert_ids) != len(np.unique(np.array(alert_ids)))
-
-    def test_alerts_in_sentinel(self):
-        """
-        This test case verifies that all Helios alerts have corresponding incidents in the Sentinel workspace.
-        """
-        alert_ids = get_alerts(self.api_key)
-        for alert_id in alert_ids:
-            self.assertIsNotNone(search_alert_id_in_incident(alert_id, self.resource_group, self.workspace_name), f"alert_id --> {alert_id} doesn't exist in sentinel.")
+        print("test_cohesity_close_helios_incident finished successfully.")
+
+
+def test_all_incidents_in_helios(self):
```

Comment:
Why is the definition test_all_incidents_in_helios and the functions below are not within the scope of the class TestCohesity?

Reply:
done